src/main/java/com/artipie/api/ManageStorageAliases.java

```diff
@@ -0,0 +1,216 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie.api;
+
+import com.amihaiemil.eoyaml.Yaml;
+import com.amihaiemil.eoyaml.YamlMapping;
+import com.amihaiemil.eoyaml.YamlMappingBuilder;
+import com.amihaiemil.eoyaml.YamlNode;
+import com.artipie.asto.Key;
+import com.artipie.asto.blocking.BlockingStorage;
+import com.artipie.misc.Json2Yaml;
+import com.artipie.misc.Yaml2Json;
+import com.artipie.settings.CrudStorageAliases;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.json.JsonObject;
+
+/**
+ * Manage storage aliases settings.
+ * @since 0.27
+ */
+public final class ManageStorageAliases implements CrudStorageAliases {
+
+    /**
+     * Settings file name.
+     */
+    public static final String FILE_NAME = "_storages";
+
+    /**
+     * Key for the settings file with .yaml extension.
+     */
+    private static final Key YAML = new Key.From(
+        String.format("%s.yaml", ManageStorageAliases.FILE_NAME)
+    );
+
+    /**
+     * Yaml storages section name.
+     */
+    private static final String STORAGES_NODE = "storages";
+
+    /**
+     * Repository or user key.
+     */
+    private final Optional<Key> key;
```

Comment:
I don't like `Optional` as a field type or an argument type.
Maybe just `Key` and use `Optional.ofNullable(this.key)`  if `Optional` is necessary?

Reply:
I strongly disagree with nullable fields, storage aliases can be sett on different levels: for one repository, for user or for whole artipie. In the last case, the key will be empty.